pkgs/development/stm32/betaflight/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub
+, gcc-arm-embedded, python2
+, skipTargets ? [
+  # These targets do not build for various unexplored reasons
+  # TODO ... fix them
+  "AFROMINI"
+  "ALIENWHOOP"
+  "BEEBRAIN"
+  "CJMCU"
+  "FRSKYF3"
+]}:
+
+let
+
+  version = "3.2.3";
+
+in stdenv.mkDerivation rec {
```

Comment:
Can you you add meta attributes, such as license, platforms, maintainers and homepage?

Reply:
Whoops, that got left behind in one of my local commits.